### DISCUSSION TITLE
Workflow for tasks without a scheduled time

### DISCUSSION THREAD

**Discussion by mariavd** [ivan-lednev/obsidian-day-planner#582] [Q&A] [answered]
I want to build a sort of kanban-style workflow where I can see a set of tasks that are open and then organise them by time and or move to another day. I use this search to see the list of tasks in my daily note,

```
not done
due before 2024-10-01
tags do not include #event/anniversary 
tags include #TODO/work
```

Is there a way to preview them in the weekly view of Day Planner? I can only see the tasks that specifically start with a time interval.

**Comment by ivan-lednev** [comment, marked as answer]
Hello. Currently there is no way to achieve this. Similar feature requests: #397
Feel free to vote on that issue or create your own feature request if that one doesn't cover your needs.

**Reply by mariavd** [reply]
I see, thank you.
